--- a/build-aux/gen-thumbnailer.py
+++ b/build-aux/gen-thumbnailer.py
@@ -21,7 +21,11 @@
 
 newenv = os.environ.copy()
 newenv['GDK_PIXBUF_PIXDATA'] = args.pixdata
-newenv['GDK_PIXBUF_MODULE_FILE'] = args.loaders
+
+# Only set if provided
+if args.loaders:
+    newenv['GDK_PIXBUF_MODULE_FILE'] = args.loaders
+
 # 'nt': NT-based Windows, see https://docs.python.org/3/library/os.html
 if os.name == 'nt':
     gdk_pixbuf_dll_buildpath = os.path.dirname(args.pixdata)
